src/components/Block.tsx

```diff
@@ -0,0 +1,53 @@
+const Block = () => {
+    return (
+        <div>
+            <div className="tds-headline-02 tds-u-pb1">Block</div>
```

Comment:
Should I add the different variants of the Block?

Reply:
Sure let's!